### PR TITLE
Initialize project with components

### PR DESCRIPTION
**Pull request by Kelvin-Ben**
-  In this project, I laid foundations for the Bookstore website. 
 - I created the React part of the app. 
 - I also set up routing using React Router.

### CODE REVIEW

File: src/components/BookFeatures.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+/* eslint-disable react/prop-types */
```
Comment (Deepakdanger) [inline src/components/BookFeatures.js:1]: - [ ] Kindly remove this & solve it manually.

File: src/components/NavBar.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable react/jsx-props-no-spreading */
```
Comment (Deepakdanger) [inline src/components/NavBar.js:2]: - [ ] Kindly remove this & solve it manually.

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @Kelvin-Ben , 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight :100: 
- No linter error :heavy_check_mark: 
- Website looks professional :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/BookFeatures.js (hunk 1 above)
Comment (Kelvin-Ben) [reply src/components/BookFeatures.js:1]: Do you have a suggestion that I can use to solve the issue?

File: src/components/NavBar.js (hunk 2 above)
Comment (Kelvin-Ben) [reply src/components/NavBar.js:2]: @Deepakdanger I have tried different ways but I was not able, that's why I opted for using eslint disabler

File: src/components/BookFeatures.js (hunk 1 above)
Comment (Deepakdanger) [reply src/components/BookFeatures.js:1]: @Kelvin-Ben , kindly check this one.[LINK](https://blog.logrocket.com/validate-react-props-proptypes/)

**Pull request closed without merging**